### PR TITLE
Add licence plugin

### PR DESCRIPTION
**Pull request by PascalSchumacher**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -286,6 +286,31 @@
                 </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
```
Comment (PascalSchumacher) [inline pom.xml]: for some reason this does not work when I add it to the plugin-management section

**Comment by PascalSchumacher** [comment]
plugin adds/updates the headers every time a maven build is run

code "stolen" from assertj ;)

**Comment by benas** [comment]
Hello Pascal,

Good idea. Until now, I update license headers manually in bulk mode with Intellij idea.

Could you please keep only the template + modification of pom file in the PR? I'll update headers afterwards.

Many thanks upfront

Cheers
Mahmoud

**Comment by PascalSchumacher** [comment]
I have dropped the update commit from the pull request.

**Comment by benas** [comment]
Thank you for the update.

**Comment by PascalSchumacher** [comment]
nice to hear that 3.1 will be released soon :+1:

**Pull request merged**